user/tests/integration/ota/min_max_app_size/min_max_app_size.cpp

```diff
@@ -42,26 +67,113 @@ void waitOta() {
     assertEqual(0, TestRunner::instance()->pushMailbox(Test::MailboxEntry().type(Test::MailboxEntry::Type::RESET_PENDING), 5000));
 }
 
+void highPriorityFunc() {
+    while (1) {
+        // Busy loop up to 5ms
+        HAL_Delay_Microseconds(random(5) * 1000);
+        // Force frequent reschedule
+        delay(1);
+    }
+}
+
+void bleScanResultCallBack(const BleScanResult *result, void *context) {
+    Log.info("BLE adv: %s", result->address().toString().c_str());
+}
+
+void wifiScanResultCallBack(WiFiAccessPoint* wap, void* context) {
+  if (apList.size() < 20)
+    apList.append(*wap);
+}
+
+void lowPriorityFunc() {
+    unsigned int wifiSeconds = 0;
+    const int WIFI_SCAN_INTERVAL_SECONDS = 10;
+
+    while (1) {
+#if HAL_PLATFORM_BLE
+        BleScanParams params = {};
+        params.version = BLE_API_VERSION;
+        params.size = sizeof(BleScanParams);
+        params.timeout = 50; // *10ms = 500ms overall duration
+        params.interval = 800; // *0.625ms = 500ms
+        params.window = 800; // *0.625 = 500s
+        params.active = false;
+        
+        BLE.setScanParameters(params);
+        BLE.scan(bleScanResultCallBack, nullptr);
```

Comment:
Why not perform the BLE.scan() in another thread?  It would allow the test to potentially run a BLE and WiFi scan is close proximity.